scan-command/src/test/resources/command-outputs/print-rules-to-console.txt

```diff
@@ -0,0 +1,13 @@
+Loading scan tool configurations from src\test\resources\test-resources\bal-project-with-config-file\Scan.toml
+	RuleID                                           | Rule Severity | Rule Description
+	------------------------------------------------------------------------------------
+	ballerina:1                                      | CODE_SMELL    | Avoid checkpanic
+	ballerina/example_module_static_code_analyzer:1  | CODE_SMELL    | rule 1
+	ballerina/example_module_static_code_analyzer:2  | BUG           | rule 2
+	ballerina/example_module_static_code_analyzer:3  | VULNERABILITY | rule 3
+	exampleOrg/example_module_static_code_analyzer:1 | CODE_SMELL    | rule 1
+	exampleOrg/example_module_static_code_analyzer:2 | BUG           | rule 2
+	exampleOrg/example_module_static_code_analyzer:3 | VULNERABILITY | rule 3
+	ballerinax/example_module_static_code_analyzer:1 | CODE_SMELL    | rule 1
+	ballerinax/example_module_static_code_analyzer:2 | BUG           | rule 2
+	ballerinax/example_module_static_code_analyzer:3 | VULNERABILITY | rule 3
```

Comment:
How is the package order decided? IMO, we should prioritize by org - ballerina, ballerinax, wso2, then other orgs alphabetically sorted or completely sort alphabetically.

I prefer the former.

Reply:
This is based on the following order, where the core rules in the scan tool are displayed first, then for the compiler plugin rules, the order is decided by the project API package resolution.

This is based on the following order:
- Initially, the core rules defined in the scan tool are shown.
- Next, the rules obtained from the compiler plugins are shown in the order determined through the package resolution.

Reply:
I would use the order I suggested. Compiler plugin order doesn't mean anything to the user, does it?